.github/dependabot.yml

```diff
@@ -0,0 +1,126 @@
+      # SmallRye
+      - dependency-name: io.smallrye:smallrye-jwt
+      - dependency-name: io.smallrye:smallrye-health
+      - dependency-name: io.smallrye:smallrye-metrics
+      - dependency-name: io.smallrye:smallrye-open-api
+      - dependency-name: io.smallrye:smallrye-opentracing
+      - dependency-name: io.smallrye:smallrye-fault-tolerance
+      - dependency-name: io.smallrye:smallrye-context-propagation
+      - dependency-name: io.smallrye.config:smallrye-config
+      - dependency-name: io.smallrye.reactive:mutiny
+      - dependency-name: io.smallrye.reactive:smallrye-reactive-messaging
+      - dependency-name: io.smallrye.reactive:smallrye-reactive-streams-operators
+      # Swagger-UI
+      - dependency-name: org.webjars:swagger-ui
+      # Tika
+      - dependency-name: org.apache.tika:tika-parsers
+      # RX Java 2
+      - dependency-name: io.reactivex.rxjava2:rxjava
+      # Test dependencies
+      - dependency-name: io.rest-assured:rest-assured
+      - dependency-name: io.rest-assured:json-schema-validator
+      - dependency-name: org.junit:junit-bom
+      - dependency-name: org.assertj:assertj-core
+      - dependency-name: org.testcontainers:testcontainers-bom
+      - dependency-name: org.testcontainers:testcontainers
+      - dependency-name: org.testcontainers:postgresql
+      - dependency-name: org.mockito:mockito-core
+      - dependency-name: org.mockito:mockito-junit-jupiter
+      - dependency-name: org.awaitility:awaitility
+      # Maven plugins
+      - dependency-name: net.revelc.code.formatter:formatter-maven-plugin
+      - dependency-name: net.revelc.code:impsort-maven-plugin
+      # Narayana
+      - dependency-name: org.jboss.narayana.jta:narayana-jta
+      - dependency-name: org.jboss.narayana.jts:narayana-jts-integration
+      - dependency-name: org.jboss.narayana.stm:stm
+      # Agroal
+      - dependency-name: io.agroal:agroal-api
+      - dependency-name: io.agroal:agroal-pool
+      - dependency-name: io.agroal:agroal-narayana
+      #Jandex
+      - dependency-name: org.jboss:jandex
+      - dependency-name: org.jboss.jandex:jandex-maven-plugin
+      #Neo4j
+      - dependency-name: org.neo4j.driver:neo4j-java-driver
+      #WireMock
+      - dependency-name: com.github.tomakehurst:wiremock-jre8
+      - dependency-name: uk.co.automatictester:wiremock-maven-plugin
+      #Picocli
+      - dependency-name: info.picocli:picocli
+      - dependency-name: info.picocli:picocli-codegen
+      # Caffeine
+      - dependency-name: com.github.ben-manes.caffeine:caffeine
+      # Jackson
+      - dependency-name: com.fasterxml.jackson:jackson-bom
+    rebase-strategy: disabled
+  - package-ecosystem: gradle
+    directory: "/integration-tests/gradle"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 3
+    labels:
+      - area/dependencies
+    rebase-strategy: disabled
+  - package-ecosystem: gradle
+    directory: "/devtools/gradle"
+    schedule:
+      interval: daily
```

Comment:
We should probably also use a schedule for these two.

That being said schedules don't seem to work well right now.

Reply:
It had the same value as the one above, I removed them on purpose, I'll put it back